spec/models/bid_spec.rb

```diff
@@ -1,5 +1,31 @@
 require 'rails_helper'
 
 RSpec.describe Bid, type: :model do
-  pending "add some examples to (or delete) #{__FILE__}"
+  describe "toggle accept!" do
+  	context "when bids have not been accepted" do
+	  	it "updates the bid to not be accepted" do
+	  		business_user = FactoryGirl.create(:business_user)
+	  		job = FactoryGirl.create(:job, agent_id: nil, client_id: business_user.id)
+	  		broker = FactoryGirl.create(:broker)
+	  		bid = FactoryGirl.create(:bid, job_id: job.id, bidder_id: broker.id)
+
+	  		expect(bid.accepted).to eq(false)
+	  		expect(bid.job.agent_id).to eq(nil)
+	  	end
+    end
+
+    context "when the bids have been accepted" do
+    	it "updates the bid to be accepted" do
+	  		business_user = FactoryGirl.create(:business_user)
+	  		job = FactoryGirl.create(:job, agent_id: nil, client_id: business_user.id)
+	  		broker = FactoryGirl.create(:broker)
+	  		bid = FactoryGirl.create(:bid, job_id: job.id, bidder_id: broker.id)
```

Comment:
In this factory definition, let's define what `accepted` is initially, so the person reading this test code knows exactly what to expect:

```
bid = FactoryGirl.create(:bid, job_id: job.id, bidder_id: broker.id, accepted: false)
```

This is a testing concept called "Mystery Guests" - https://robots.thoughtbot.com/mystery-guest

The gist is that for tests, you shouldn't try to make the code DRY, you should focus more on readability and understandability :)

Since the specs serve as specifications to the application, you should make it clear as possible, even for factory definitions :)